fr-portfolio.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en-gb">
+<head>
+ <link rel="stylesheet" type="text/css" href="normalize.css">
+ <link rel="stylesheet" type="text/css" href="portfolio.css">
+ <link rel="stylesheet"     href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+ <link rel="preconnect" href="https://fonts.googleapis.com">
+ <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+ <link href="https://fonts.googleapis.com/css2?family=Comfortaa&display=swap" rel="stylesheet">
+ <title>Freddie's Portfolio</title>
+</head>
+<body>	
+ <main>	
+  <nav class="navbar">
+   <a href="#portfolio">Portfolio</a>
+   <a href="#about">About</a>
+   <a href="#contact">Contact</a>
+  </nav>
+	
+  <section class="welcome-banner" > 
+   <h1>Freddie's Portfolio</h1>
+  </section>	<!--.welcome-banner -->
+	
+  <article id="about" class="about-alt">
+   <h2> About me </h2>
+   <figure class="about-me-container">
+    <img src="https://via.placeholder.com/350x200" alt="gray square as placeholder">
+    <p>Hi this is where I should put some information about myself</p>
+   </figure>	<!--.about-me-container -->	
+  </article> <!--#about --> <!--.about-alt -->
+		
+  <section id="portfolio" class="projects"> 
+   <h2>Portfolio</h2>
+   <div class="portfolio-container"> 
+    <figure>
+     <img src="https://via.placeholder.com/350x200" alt="gray square as placeholder">
+     <p>This is where my project information will go</p>
+    </figure>
+   </div> <!--.portfolio-container -->
+   <div class="portfolio-container">
+    <figure>
+     <img src="https://via.placeholder.com/350x200"  alt="gray square as placeholder">
+	 <p>This is where my project information will go </p>
+    </figure>	
+   </div>  <!--.portfolio-container -->
+   <div class="portfolio-container">	
+    <figure>
+     <img src="https://via.placeholder.com/350x200"  alt="gray square as placeholder">
+     <p>This is where my project information will go This is where my project information will go This is where my project information will go</p>
+    </figure>
+   </div> <!--.portfolio-container -->
+   <div class="portfolio-container">	 	
+    <figure>
+     <img src="https://via.placeholder.com/350x200"  alt="gray square as placeholder">
+     <p>This is where my project information will go</p>
+    </figure>
+   </div>	<!--.portfolio-container -->
+  </section> <!--.projects --> <!--#portfolio -->
+  
+  <footer id="contact" class="contact-me">
+   <h2>Contact</h2>
+   <div class="my-contacts">
+    <div>
+     <h3>Email</h3>
+     <i class="fas fa-at"></i>
+    </div>
+    <div>
+     <h3>LinkedIn</h3>
+     <i class="fab fa-linkedin"></i>
+    </div>
+    <div>
+     <h3>Github</h3>
+     <i class="fab fa-github"></i>
+    </div>	
```

Comment:
these icons are supposed to be links - wrap each one in a <a>

Reply:
Think I've done it